scripts/wredstat

```diff
@@ -0,0 +1,414 @@
+#!/usr/bin/env python3
+
+#####################################################################
+#
+# wredstat is a tool for summarizing wred queue statistics of all ports.
+#
+#####################################################################
+
+import json
+import argparse
+import datetime
+import os.path
+import sys
+
+from collections import namedtuple, OrderedDict
+from natsort import natsorted
+from tabulate import tabulate
+from sonic_py_common import multi_asic
+
+# mock the redis for unit test purposes #
+try:
+    if os.environ["UTILITIES_UNIT_TESTING"] == "2":
+        modules_path = os.path.join(os.path.dirname(__file__), "..")
+        tests_path = os.path.join(modules_path, "tests")
+        sys.path.insert(0, modules_path)
+        sys.path.insert(0, tests_path)
+        import mock_tables.dbconnector # lgtm [py/unused-import]
+    if os.environ["UTILITIES_UNIT_TESTING_TOPOLOGY"] == "multi_asic":
+        import mock_tables.mock_multi_asic
+        mock_tables.dbconnector.load_namespace_config()
+
+except KeyError:
+    pass
+
+from swsscommon.swsscommon import SonicV2Connector
+from utilities_common.cli import json_serial, UserCache
+from utilities_common import constants
+import utilities_common.multi_asic as multi_asic_util
+
+QueueStats = namedtuple("QueueStats", "queueindex, queuetype, wredDrppacket, wredDrpbytes, ecnpacket, ecnbytes")
+header = ['Port', 'TxQ', 'WredDrp/pkts', 'WredDrp/bytes', 'EcnMarked/pkts', 'EcnMarked/bytes']
+voq_header = ['Port', 'Voq', 'WredDrp/pkts', 'WredDrp/bytes', 'EcnMarked/pkts', 'EcnMarked/bytes']
+
+counter_bucket_dict = {
+    'SAI_QUEUE_STAT_WRED_DROPPED_PACKETS': 2,
+    'SAI_QUEUE_STAT_WRED_DROPPED_BYTES': 3,
+    'SAI_QUEUE_STAT_WRED_ECN_MARKED_PACKETS': 4,
+    'SAI_QUEUE_STAT_WRED_ECN_MARKED_BYTES': 5,
+}
+wred_drop_pkts_capable = False
+wred_drop_bytes_capable = False
+ecn_marked_pkts_capable = False
+ecn_marked_bytes_capable = False
+
+from utilities_common.cli import json_dump
+from utilities_common.netstat import ns_diff, STATUS_NA
```

Comment:
Is there any reason why these import statements are not placed along with other imports?

Reply:
I have fixed this. Thank you for pointing it out..!